client/plasmacli/query/main.go

```diff
@@ -12,3 +17,35 @@ var queryCmd = &cobra.Command{
 func QueryCmd() *cobra.Command {
 	return queryCmd
 }
+
+func OutputInfo(ctx context.CLIContext, pos plasma.Position) (store.OutputInfo, error) {
```

Comment:
dup logic here, but should be replaced in favor of TransactionOutput(), maybe we should add a `query.go` file and put all queries in there